packages/compass-generative-ai/src/components/ai-optin-modal.tsx

```diff
@@ -104,25 +81,34 @@ export const AIOptInModal: React.FunctionComponent<OptInModalProps> = ({
   }, [track, onOptInModalClose]);
 
   return (
-    <ConfirmationModal
+    <MarketingModal
+      className={css({
+        zIndex: '99999',
```

Comment:
The zIndex tells me there's a visual conflict with another modal.

Slightly better: zIndex: String(otherZIndex + 10) to make that relationship clearer
Better: No two modals render at the same time